src/applications/simple-forms/21-0966/config/helpers.js

```diff
@@ -288,6 +290,35 @@ export const confirmationPageAlertHeadline = formData => {
   return 'You’ve submitted your intent to file';
 };
 
+const formatDate = date => {
+  if (date) {
+    return format(new Date(date), 'MMMM d, yyyy');
+  }
+  return '';
+};
+
+export const confirmationPageAlertHeadlineV2 = ({
```

Comment:
you can compare with `confirmationPageAlertHeadline` right above this. This is V2